main_app/templates/coach/detail.html

```diff
@@ -15,10 +15,12 @@ <h1>{{ coach.name }}</h1>
         <p>Specialty: {{ coach.specialty }}</p>
         <p>Schedule/Availability: {{ coach.availability }}</p>
         </div>
+        {% if perms.main_app.add_business %}
```

Comment:
I'm fine with leaving perm lock as "add_business" even though this is for coach instead.

Functionally, it does the same thing. Only an upgraded account can do XYZ, including any and all business crud.

Leave as is?

Reply:
yeah i realized that as-well when i was trying to test the app. I wasn't sure how to add the 'coach' permissions when the account is upgraded so i worked with what you already had set up. after adding the perm lock the functionality seemed to be the same so i think its fine to just leave it as it .

Reply:
can be changed later if need be. Will approve now as functionality is working as intended! Thanks!